content/zh-tw/shift-left.md

```diff
@@ -0,0 +1,34 @@
+---
+title: 向左移
+status: Completed
+category: 概念
+tags: ["方法論", "", ""]
+---
+
+## 是什麼 {#what-it-is}
+
+如果我們將將軟體開發週期想像成一條從左到右依序執行的線，
+「向左移」中的「左」指的便是週期中較早的階段。
+向左移是在軟體開發生命週期中的早期階段就進行測試、安全性等其他開發活動，
+而避免等到週期的末期才進行的一種實踐。
+
+儘管最初是用來指稱早期測試的過程，
+但現在向左移也可以應用於軟體開發和 [DevOps](/zh-tw/devops/) 的其他方面，如安全性和部署。
+
+## 解決的問題 {#problem-it-addresses}
+
+如果在開發週期後期或部署後才發現安全問題、錯誤和軟體缺陷，
+修復它們可能會變得更加困難且昂貴，尤其是如果該軟體已經部署到產品環境中。
+
+## 如何幫助 {#how-it-helps}
+
+透過在軟體開發中採用向左移的思維，團隊可以在整個開發生命週期中進行測試和加強安全性。
+由於測試和安全性的責任由開發團隊共同承擔，從軟體工程師到品質保證再到維運，每個人都在確保應用程式的穩定性和安全性。
```

Comment:
```suggestion
透過在軟體開發中採用向左移的思維，
團隊可以在整個開發生命週期中進行測試和加強安全性。
由於測試和安全性的責任由開發團隊共同承擔，
從軟體工程師到品質保證再到維運，
每個人都在確保應用程式的穩定性和安全性。
```

It is recommended to the new line to follow the English upstream and it is easy to maintain when the subsequent English upstream changes.
In accordance with the [style guide](https://glossary.cncf.io/style-guide/#three-subheadings) set forth by the cncf/glossary,please use semantic line breaks (one sentence per line)